README.md

```diff
@@ -165,23 +175,85 @@ In Device Simulator Express, you can use keyboard to interact with the device:
+1. Type in `"Device Simulator Express: [Circuit Playground Express] New File"` in the command palette (`CTRL + SHIFT + P` / `CMD + SHIFT + P` to open the command palette).  
+<img alt="Deploy to Device" src='assets/readmeFiles/getting_started.png'>
+2. Choose the `micro:bit` option from the dropdown.
+2. Read, copy and learn some of the things you can do with the simulator!
+   
+#### II. Start with the "Device Simulator Express [micro:bit]: New File" Command.
+
+1. Type in `"Device Simulator Express: [micro:bit] New File"` in the command palette (`CTRL + SHFT + P / CMD + SHIFT + P` to open the command palette).  
+   <img alt='"New File" animation' src='assets/readmeFiles/microbit/microbit-new-file.gif'>
+2. Name and save your file somewhere, and we’re good to go!
+
+#### III. Start from an existing python file.
+
+1. Open the folder or your .py file in Visual Studio Code.
+2. Run `Device Simulator Express: [micro:bit] Open Simulator` from the command palette or icon in the editor toolbar.
+
+#### IV. Run your code on the simulator.
+
+<img alt='How to run the simulator animation' src='assets/readmeFiles/microbit/microbit-run.gif'>
+
+1. Run `Run Simulator` from the command palette or use the `Play` button on the simulator webview.
+
+#### V. Deploy your code to the physical device
+
+1. Run `[micro:bit] Deploy to Device` from the command palette
+
+<img alt="Deploy to Device" src="assets/readmeFiles/microbit/microbit-deploy.png">
+
+#### VI. Use the Serial Monitor for your BBC micro:bit device (available on Windows and Mac only)
+
+1. Plug in your BBC micro:bit.
+2. Run the command `"Device Simulator Express: Open Serial Monitor"`.
+3. Select your baud rate for the serial port.
+4. The `print()` statements in your code will show in the output console.
+
+#### VII. Use the sensors in the Device Simulator Express
+
+Generating input for the sensors can be done by interacting directly with device on the webview
+or by using the toolbar.
+
+- **Push buttons:** click directly on the buttons on the device or use the keybindings.
+- **Temperature sensor, Light sensor, Acceleration sensor:** click on the corresponding button in the toolbar and change the value using the slider or the input box attached to it.
+
+#### VIII. Debug your project on the simulator
+
+1. Add breakpoints in your code
+2. Press F5 to enter the debugging mode, and you can start debugging line by line!
+
+### Commands
 Using the simulator for the micro:bit is similar to using the one for the CPX. The only difference is that the commands in the command palette display `Device Simulator Express: [micro:bit] <command>` instead of `Device Simulator Express: [Circuit Playground Express] <command>`. Currently, we support the following commands for micro:bit:
+- `Device Simulator Express: Getting Started`: Opens a page in VS Code that helps users get started with the extension and code they can write to play with the simulators.

```

Comment:
This is a bit hard to understand? Maybe go with "Opens a page in VS Code that helps users get started with the extension. Here, users can browse through code that they can use to play with the simulators."

Reply:
nice ill change that